plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -57,7 +57,7 @@
     type: bool
   database:
     description:
-      - The database to target. Used by the 'createcachetable', 'flush', 'loaddata', and 'syncdb' commands.
+      - The database to target. Used by the 'createcachetable', 'flush', 'loaddata', 'syncdb', and 'migrate' commands.
```

Comment:
```suggestion
      - The database to target. Used by the C(createcachetable), C(flush), C(loaddata), C(syncdb), and C(migrate) I(command).
```

Reply:
@pawl could you fix this? then we'll merge

Reply:
@Andersson007 Done. Thanks, good to know about the [formatting-functions](https://docs.ansible.com/ansible/2.5/dev_guide/developing_modules_documenting.html#formatting-functions).